lib/data_migrator/offence_code_generator.rb

```diff
@@ -34,6 +34,15 @@ def scheme_10_offence_unique_code
     description.abbreviate +
```

Comment:
not a blocker - ^^ line above, for clarity this could be named `scheme_10plus_offence_unique` or similar